src/property-filter/internal.tsx

```diff
@@ -187,7 +187,7 @@ const PropertyFilterInternal = React.forwardRef(
           break;
         }
       }
-      if (internalFreeText.disabled && !('propertyKey' in newToken)) {
+      if (internalFreeText.disabled && !newToken.property) {
```

Comment:
The token was replaced with an internal token so that the "propertyKey" was replaced with "property".